src/Nethermind/Nethermind.Specs/Forks/18_Prague.cs

```diff
@@ -20,6 +20,7 @@ protected Prague()
         IsEip7002Enabled = true;
         IsRip7212Enabled = true;
         Eip2935ContractAddress = Eip2935Constants.BlockHashHistoryAddress;
+        IsEip4844PectraEnabled = true;
```

Comment:
This is for Gnosis only, so shouldn't be in mainnet related class, we can try enabling it somewhere through `GnosisSpecProvider`

Reply:
Makes sense, however the change does actually affect mainnet as it means that burnt blob fees are reported in tracing after the transition. Do you think I should separate this out so the tracing is fixed even before the transition?

Reply:
I think this is just a fix to tracing then or we should change tracer so it accepts 2 types of burnt fees

Reply:
Ok I did this by having two instances in `Prague`, what do you think?

Reply:
Just use `OverridenSpec` in `GnosisSpecProvider` and set this EIP. In prod we still just parse JSON so it is mostly for tests.